### DISCUSSION TITLE
Idea: different mobile views of sidebar

### DISCUSSION THREAD

**Discussion by axtg** [buefy/buefy#3994] [Ideas]
I have been considering moving my top navigation to the side, much like the Buefy documentation navigation (left sidebar). But I am struggling with the mobile view: 

1. Option 1 move it down, like at Buefy's docs. Pro: no additional interactions needed besides scrolling. Con: requires scrolling of the user which is painful on long pages.
2. Option 2 reduce it to be smaller, like it's current behaviour on mobile (when preferred). Pro: always visible. Cons: icons can be unclear, screen estate is lost.
3. Option 3 (new): make it behave (on mobile) like a dropdown, move content to modal. Pro: clean, no loss of screen estate unless needed. Con: requires click, position might change
4. Option 4 (new): roll-up. Make it behave like an accordion. Click to expand. Pro: clean, no loss of screen estate, common behaviour. Con: requires a click, element might get lost when scrolling.

Any other ideas? Do people recognize this struggle? Other solutions perhaps?

**Comment by jtommy** [comment]
They are good idea but I don't understand well the last option; what would you like to see on mobile?

**Comment by axtg** [comment]
> They are good idea but I don't understand well the last option; what would you like to see on mobile?

For 4 I meant something similar to the Collapse / Card.

Obviously, and that might be my next best step, I can do all these things by triggering the sidebar from any element. A floater, a button, navbar. I'd just be curious to learn how others tackle the challenge. Including the team here. I can imagine that the current navigation for the documentation on a mobile view has been reconsidered...? 

And then maybe we can come up with a default best to evolve the sidebar with eg an additional mobile view besides the current 'reduced' view. 

Anyway, here to exchange thoughts 😊.

**Comment by jtommy** [comment]
The option 3 could be interesting but the option 4 is already available playing a little with open and mobile hidden

**Comment by Jefwillems** [comment]
Also the can-cancel option 'x' doesn't seem to be doing anything? At least it’s not clear from the docs

**Comment by jtommy** [comment]
@Jefwillems it's wrong, i'm removing it from doc

**Comment by Jefwillems** [comment]
Thanks
